client/app/scripts/superdesk-ingest/module.js

```diff
@@ -1358,7 +1358,11 @@ define([
                     {action: 'list', type: 'ingest'}
                 ],
                 privileges: {fetch: 1},
-                key: 'f'
+                key: 'f',
+                additionalCondition: ['desks', function (desks) {
+                    // fetching to 'personal' desk is not allowed
+                    return (desks.getCurrentDeskId() !== 'personal');
+                }]
```

Comment:
I've seen in the linked examples that `condition` is a function that accepts the `item`parameter, while the `additionalCondition` can accept arbitrary dependencies specified by the Angular-like array syntax. I hence used the latter, is that fine?